### PR TITLE
Nv 899 pagination display and styles

### PR DESCRIPTION
**Pull request by BhuvaneshPatil**
### What change does this PR introduce? 
- Adds condition in `getPageCount()` method to handle the count when number of records are less than pagesize.
- Adds styling to pagination component so that it's at the bottom of the page.

### Why was this change needed?

#2194 

### Other information (Screenshots)
image

### CODE REVIEW

File: apps/web/src/design-system/table/Table.tsx [hunk 1]
```diff
@@ -89,15 +88,20 @@ export function Table({
   const handlePageChange = (pageNumber) => {
     gotoPage(pageNumber - 1);
   };
+
   const getPageCount = () => {
+    if (total <= pageSize) {
+      return 0;
+    }
+
```
Comment (scopsy) [inline apps/web/src/design-system/table/Table.tsx:96]: A small question just wondering, why is this needed if on line 152 we are checking:
`getPageCount() > 1`

Isnt it will only display if there is more than total pages?

Comment (BhuvaneshPatil) [reply apps/web/src/design-system/table/Table.tsx:96]: Yes. Sorry didn't noticed it, after all they are doing same work.

**Comment by scopsy** [comment]
@BhuvaneshPatil looks like you worked against main branch :( 

Could you please recreate this PR from the `next` branch since looks like we have her too much conflicts. Will close this for now
